### ISSUE TITLE
When clicking on floating searchbar select the entire URL

### ISSUE THREAD

**Issue by MordechaiHadad** [zen-browser/theme-components#16]
Reference:

https://github.com/user-attachments/assets/7e4232fd-6a60-4916-8b28-7beb59fc2dd4

**Comment by alvindera97** [comment]
Yup... I agree, I'm probably waaay too accustomed to just clicking on the address bar and typing straight away without having to select all text again. It will be a nice to have if this is implemented.

I had to uninstall the floating address bar theme because of this.. I'll reinstall once it's implemented.

**Comment by samgurung** [comment]
> Yup... I agree, I'm probably waaay too accustomed to just clicking on the address bar and typing straight away without having to select all text again. It will be a nice to have if this is implemented.
>
I second this.
 
> I had to uninstall the floating address bar theme because of this.. I'll reinstall once it's implemented.

Same her..uninstalled waiting for the feature to be implemented

**Comment by jimfr** [comment]
I third this

**Comment by Anthonyy232** [comment]
fourthed

**Comment by SakibShahariar** [comment]
5th this

**Comment by enoversum** [comment]
Sixing the hell out of this. Thought this was a Zen browser bug, yet it appears to be this Mod.

**Comment by JacksonZmy** [comment]
6

**Comment by shanto** [comment]
7

**Comment by ell1010** [comment]
weird that it doesnt do it anyway

**Comment by Li-An** [comment]
Same here. I often click on bar and copy URL. It’s impossible for the moment.

**Comment by ChrisToxz** [comment]
Yes I agree, I really miss this behavior! 
Disabled for the time being, because it really kills the UX to me, can't wait to get it added!

**Comment by yab-codes** [comment]
add it already pleeaase. disabling the mod until the feature its added

**Comment by metafii** [comment]
That's exactly what it is. I won't use it until this feature comes. 
Edit: If you open the search bar with Ctrl+L, this error does not occur and I now use abbreviations with this theme mode :D 
thx dev team

**Comment by cboehm-it** [comment]
Yeah I would really like this change.
@metafii Thank you very much for this hint.

**Comment by martian0x80** [comment]
Alt+D and Ctrl+L works for now, but the bar defaulting to this would be great, also there's a weird behavior where after hovering the mouse over the url select that portion of the text from the left (from the left end to the current position) like I am holding the left click (but I am not).

Maybe, we could also have the option to select a certain portion of the text in the url bar by selecting it, and the floating url bar opens up after the mouseup event is fired. But, that's an entirely different issue.
